app/code/community/Bolt/Boltpay/Model/BoltOrder.php

```diff
@@ -752,6 +752,12 @@ public function getBoltOrderToken($quote, $checkoutType)
                 return json_decode('{"token" : "", "error": "'.$this->boltHelper()->__('Billing address is required.').'"}');
             }
         }
+        if ($isMultiPage && !$quote->validateMinimumAmount()) {
```

Comment:
Only for multipage? Wouldn't minimum amount also apply to payment only as well?

Reply:
@adentong 
For payment only, the Bolt Button couldn't be reached or wouldn't be rendered until the cart is in a valid condition.  We only have to use this sort of front-end validation when Bolt is bypassing the native Magento interface that already accounts for this.

Reply:
Got it.